core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.html

```diff
@@ -111,14 +111,9 @@
           <div class="oppia-outer-container">
             <div class="oppia-mascot-container">
               <div class="oppia-mascot-avatar-container">
-                <a [ngClass]="{'oppia-disabled-image-link': (OPPIA_AVATAR_LINK_URL === null)}"
-                   [href]="OPPIA_AVATAR_LINK_URL"
-                   rel="noopener"
-                   target="_blank">
-                  <img class="oppia-mascot-avatar"
-                       [src]="OPPIA_AVATAR_IMAGE_URL"
-                       alt="Oppia Mascot Avatar">
-                </a>
+                <img class="oppia-mascot-avatar"
```

Comment:
Can you show a proof of these changes. You may go to the Translate Text tab in the contributor dashboard and see.